python/packages/devui/pyproject.toml

```diff
@@ -19,6 +19,7 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
   "Programming Language :: Python :: 3.13",
+  "Framework :: Pydantic :: 2",
```

Comment:
remove this, as we are no longer building on it for everything, it is a minor dependency now